scene/main/node.h

```diff
@@ -33,6 +33,7 @@
 
 #include "core/string/node_path.h"
 #include "core/templates/map.h"
+#include "core/templates/oa_hash_map.h"
```

Comment:
You included `OAHashMap` but you used `HashMap`

Reply:
Right thanks!